### PR TITLE
#122 Unit test for the restrictions-alert fix

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-commons-pluginus/issues/122 |
| Decisions | • Added the test "Closes on Undo" for the `restrictions-alert` component. The test goes as follows: <br>&nbsp;&nbsp;&nbsp;&nbsp;- Triggers event "restrictions" with changes to open the `restritions-alert` <br>&nbsp;&nbsp;&nbsp;&nbsp;- Triggers the event "undo" which will make the `restritions-alert` close<br>• Added the test "'Doesn't open when an "restrictions" event, with no changes, is triggered'" for the `restrictions-alert` component. The test goes as follows: <br>&nbsp;&nbsp;&nbsp;&nbsp;- Triggers event "restrictions" with no changes <br>&nbsp;&nbsp;&nbsp;&nbsp;- The `restritions-alert` shouldn't open after the event |
| Animated GIF | tests_passing |

### CODE REVIEW

File: vue/test/components/restrictions-alert/restrictions-alert.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("Restrictions Alert", () => {
+    it("Closes on Undo", () => {
```
Comment (gcandal) [inline vue/test/components/restrictions-alert/restrictions-alert.test.js:5]: Add a similar test for when the restrictions are triggered but also empty.

File: vue/test/base.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const localVue = testUtils.createLocalVue();
 localVue.use(plugins.busPlugin);
 localVue.use(components.install);
 localVue.component("global-events", globalEvents.default);
+localVue.filter("locale", mocks.locale);
```
Comment (joamag) [inline vue/test/base.js:14]: create an install method in the mock directory
